client/html/templates/common/summary/detail-body-default.php

```diff
@@ -201,6 +201,20 @@
 						<a class="product-name" href="<?php echo $enc->attr( $this->url( $detailTarget, $detailController, $detailAction, $params, array(), $detailConfig ) ); ?>">
 <?php		echo $enc->html( $product->getName(), $enc::TRUST ); ?>
 						</a>
+
+						<p class="code">
+							<span class="name">
+							<?php
```

Comment:
Just a minor improvement to be aligned to the other code: Please either put it in one line like `<span ...><?php ... ?></span` or remove the white space before `<?php ...`